include/mpm_explicit.tcc

```diff
@@ -537,3 +537,249 @@ void mpm::MPMExplicit<Tdim>::write_vtk(mpm::Index step, mpm::Index max_steps) {
   }
 }
 #endif
+
+//! MPM Explicit solver
+template <unsigned Tdim>
+bool mpm::MPMExplicit<Tdim>::solve() {
+  bool status = true;
+  
+  // Get anlysis type USL/USF
+  bool USL = false;
+  if (io_-> analysis_type() == "MPMExplicitUSL2D"|| io_-> analysis_type()  == "MPMExplicitUSL3D")
```

Comment:
Please run clang-format to format your code, otherwise it will fail in CircleCI continous integration test.